fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/AbstractEventCreationFragment.kt

```diff
@@ -202,7 +202,9 @@ abstract class AbstractEventCreationFragment : Fragment() {
     startDateEditText?.setText("")
     startDate = null
 
-    if (compareWithNowByDay(newDate) < 0) {
+    // let the comparison go to -1 for cases where the time is just after midnight
+    // this is handled just fine by computeTimeInSeconds() as an Instant also contains the date
+    if (compareWithNowByDay(newDate) < -1) {
```

Comment:
Refactor opportunity: this check appears twice (here and line 224), so it'd be better to extract it into a variable

Reply:
Yes, thanks for spotting that, I'll correct it